assets/js/modules/analytics/util/data-mock.js

```diff
@@ -25,6 +25,7 @@ import castArray from 'lodash/castArray';
 import { zip, from, Observable } from 'rxjs';
 import { map, reduce, take } from 'rxjs/operators';
 import { MODULES_ANALYTICS } from '../datastore/constants';
```

Comment:
@sashadoes Please move the `MODULES_ANALYTICS` and `stringToDate` imports within `Internal dependencies`